I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 909, in script
    #Scene 3: Meeting Darren in the library to help him write his speech
ScriptError: could not find label 'route3_2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 909, in script
    #Scene 3: Meeting Darren in the library to help him write his speech
  File "C:\Users\Anthony G\Downloads\Games\Renpy\renpy-7.3.2-sdk\renpy\ast.py", line 1695, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\Anthony G\Downloads\Games\Renpy\renpy-7.3.2-sdk\renpy\script.py", line 894, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'route3_2'.

Windows-8-6.2.9200
Ren'Py 7.3.2.320
The Death Contract 0.9
Sat Jul 20 16:46:46 2019
